src/Control/Director.php

```diff
@@ -108,7 +108,6 @@ class Director implements TemplateGlobalProvider
 
     public function __construct()
     {
-        $this->constructExtensions();
```

Comment:
Does this change mean that Director is no longer extensible, or that is doesn't need to call constructExtensions any more?

Reply:
It doesn't need it.